README.md

```diff
@@ -10,55 +10,84 @@ for most output definitions.
 ## Getting started
 
 ### Prerequisites
-* Python 3.8 installed
+
+* Python 3.10 installed
 * If the installation (pip install) is executed behind a (corporate) proxy, the following environments variables must be set: `http_proxy` and `https_proxy` (including authentication e.g., `http://${proxy_username):$(proxy_password)@yourproxy.yourdomain`)
-* If you do not run with administration rights, you may need to configure pip target path to write to your user home directory or consider [using the `pipenv` method](#setup-with-pipenv).
+* If you do not run with administration rights, you may need to configure pip target path to write to your user home directory or consider using one of the `virtual environment` methods.
 
-```
-On Unix and Mac OS X the configuration file is: $HOME/.pip/pip.conf
-If the file does not exist, create an empty file with that name.
+### Project Setup
 
-Add or replace the following lines:
-[global]
-target=/somedir/where/your/account/can/write/to
+* If you use a custom pip installation directory, set the `PYTHONPATH` environment variable to the directory that you set in the `pip.ini` file.
 
-On Windows, the configuration file is: %HOME%\pip\pip.ini
-If the file does not exist, create an empty file with that name.
+### Setup with `virtualenv`
 
-Add or replace the following lines:
-[global]
-target=C:\SomeDir\Where\Your\Account\Can\Write\To
+```sh
+python3 -m venv venv
```

Comment:
I guess the same way as for `pipenv local` we need to add here `source venv/bin/activate`.

Reply:
Fixed.